test/bin/ci_phase_test.sh

```diff
@@ -9,6 +9,9 @@ source "${SCRIPTDIR}/common.sh"
 
 cd "${TESTDIR}"
 
+# Start the web server to host ostree commit repository.
+bash -x ./bin/start_webserver.sh
```

Comment:
The server should have been started in the boot phase script. This will restart it, which should not break anything, but I wonder if you ran into a problem where it wasn't running?

Reply:
Yes, because of https://github.com/openshift/microshift/blob/main/test/bin/ci_phase_iso_boot.sh#L50
Since these are `ci_phase_*.sh` scripts, we could just keep first `start_webserver.sh` in iso_boot and let it run indefinitely

Reply:
Eventually I would like to merge the phases. There's no reason to run the scripts from different pods on the CI system, and I think it will be simpler. Now that we have it working, though, I think that sort of change is a bit of a distraction, so let's just keep this change.